website/static/js/pages/project-dashboard-page.js

```diff
@@ -122,9 +122,8 @@ $(document).ready(function () {
         currentUserCanEdit: window.contextVars.currentUser.canEdit
     });
     var newComponentElem = document.getElementById('newComponent');
-    if (newComponentElem) {
-        m.mount(newComponentElem, AddComponentButton);
-    }
+
+    m.startComputation();
```

Comment:
With the institutions mount moved inside the isRetracted block, the start Computation can also be moved inside the isRetracted block (see next comment)